core/scripts/handle.py

```diff
@@ -0,0 +1,108 @@
+"""
+Handle handles creating and updating WCosa projects
+"""
+
+from shutil import copyfile
+from colorama import Fore
+from core.scripts.others.output import write, writeln
+from core.scripts.others import helper
+import core.scripts.templates.config as config
+import core.scripts.templates.cmake as cmake
```

Comment:
our imports should probably start with `wcosa` instead of `core`, also the `scripts` qualifier seems a little redundant.

We want to be able to run `import wcosa` in the python interpreter essentially anywhere

Reply:
sure will look into that